packages/react-components/src/components/MessageStatusIcon.tsx

```diff
@@ -28,7 +28,7 @@ export const MessageStatusIcon = (props: MessageStatusIconProps): JSX.Element =>
   return (
     <>
       {/* live message is used here so that aria labels are announced on mobile */}
-      {ariaLabel && <LiveMessage message={ariaLabel} ariaLive="polite" />}
+      {ariaLabel && shouldAnnounce && <LiveMessage message={ariaLabel} ariaLive="polite" />}
```

Comment:
It looks like because of this change, message status is not announced for any messages sent after the first one (without deleting any messages)
Steps:
- send 1 message
- send a few more mesages and observe

Reply:
I've narrowed this issue down to Edge and Chrome.
Firefox and Safari do not have the issue.
I've also raised an issue through Edge's feedback system.

Reply:
Tested, after the change to `ariaLive="polite"`, Safari started to work for me again